app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elfs: list[Elf]) -> list:
+    [elf.play_elf_song() for elf in elfs]
```

Comment:
use for loop, no need to create this list. I udnerstand the idea, but it's bad for explicity, same with dwarves.